src/app/features/tech-record/components/tech-record-title/tech-record-title.component.ts

```diff
@@ -74,4 +78,13 @@ export class TechRecordTitleComponent implements OnInit {
           : this.router.navigateByUrl(`/tech-records/${this.vehicleTechRecord?.systemNumber}/change-vta-visibility`)
       );
   }
+  navigateToChangeVehicleType() {
+    this.currentTechRecord$
+      .pipe(take(1))
+      .subscribe(techRecord =>
+        techRecord?.statusCode === StatusCodes.PROVISIONAL
+          ? this.router.navigateByUrl(`/tech-records/${this.vehicleTechRecord?.systemNumber}/provisional/changeVehicleType`)
```

Comment:
have we camel cased other URLs? Genuine question as I don't know but feels odd

Reply:
ohhhh yeah... fixed